portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,69 @@
+                <h2 class="card-text">High School</h2>
+                <p class="card-text">I was born and raised in the Bay Area, and I attended Palo Alto High School.  In high school, I founded two local clubs, Coding Trek and Silicon Valley Star Trek Club, and I was featured in four news articles.</p>
+                <h3 class="card-text">Coding Trek</h3>
+                <h4 id="text" class="card-text"></h4>
+                <p class="card-text">As the founder of Coding Trek, I designed a computer science curriculum for beginners and organized a series of free coding workshops for all ages.  My club was featured in <a href="https://www.losaltosonline.com/news/sections/schools/213-school-noteworthies/56854-" target="_blank">Los Altos Town Crier</a>.</p>
+                <h3 class="card-text">Silicon Valley Star Trek Club</h3>
+                <marquee><h4 class="card-text">To Boldly Go Where No One Has Gone Before</h4></marquee>
+                <p class="card-text">As the founder of Silicon Valley Star Trek Club, I organized weekly discussions about artificial intelligence, utopia, diversity, equality, justice, humanity, and technology.  I also collected and compiled 300 Star Trek quotes.  My club was featured in <a href="http://www.mercurynews.com/2017/04/26/star-trek-travels-well-even-to-the-next-generation/" target="_blank">The Mercury News</a>, <a href="https://www.paloaltoonline.com/news/2017/04/22/palo-alto-teen-creates-her-own-trekkie-starship-crew" target="_blank">Palo Alto Online</a>, and <a href="http://www.losaltosonline.com/news/sections/community/177-features/54219-teen-hopes-star-trek-club-lives-long-and-prospers" target="_blank">Los Altos Town Crier</a>.</p>
+                <h2 class="card-text">Hobbies</h2>
+                <ul class="list-group list-group-flush">
+                    <li class="list-group-item list-group-item-light">Jigsaw Puzzles</li>
+                    <li class="list-group-item list-group-item-light">Piano 🎹</li>
+                    <li class="list-group-item list-group-item-light">Swimming 🏊‍</li>
+                    <li class="list-group-item list-group-item-light">Badminton 🏸</li>
+                    <li class="list-group-item list-group-item-light">Table Tennis 🏓</li>
+                    <li class="list-group-item list-group-item-light">Fencing 🤺</li>
+                    <li class="list-group-item list-group-item-light">Rock Climbing 🧗‍</li>
+                </ul>
+            </div>
+        </div>
+    </main>
+    <footer>
+        <form action="/data" method="POST">
+            <div class="form-group">
+                <label for="comment">Comment</label>
+                <textarea class="form-control" name="comment" cols="50" rows="10"></textarea>
+                <br>
+                <input class="btn btn-primary" type="submit" value="Submit">
+            </div>
+        </form>
+        <p>Comments</p>
```

Comment:
Maybe you could add a no comments message if there aren't any yet? Or hide this `<p>` if there aren't any comments? 

If you make that change, I suggest doing it in a following PR, there's enough going on in this one :)